extensions/mybatis/runtime/src/main/java/io/quarkus/mybatis/runtime/MyBatisRuntimeConfig.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.mybatis.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "mybatis")
+public class MyBatisRuntimeConfig {
+
+    /**
+     * MyBatis environment id
+     */
+    @ConfigItem(defaultValue = "quarkus")
+    public String environment = "quarkus";
```

Comment:
No need to explicitly set the value here. The `defaultValue` in the `@ConfigItem` annotation will already set it for you